src/info.rs

```diff
@@ -74,10 +76,36 @@ impl std::fmt::Display for Info {
             )?;
         }
         if !self.disable_fields.project {
-            write_buf(
-                &mut buf,
-                &self.get_formatted_info_label("Project: ", color),
-                &self.project_name,
+            let mut branches_tags_str: String = String::new();
+            let mut tags_str: String = String::new();
+            let mut branches_str: String = String::new();
+
+            if self.number_of_branches == 1 {
+                branches_str = String::from("1 branch");
+            } else if self.number_of_branches > 1 {
+                branches_str = format!("{} branches", self.number_of_branches);
+            }
```

Comment:
This allocates memory for a `String`, then potentially creates a new `String`.
if-else statements can be expressions. I think the more "rusty" way of doing this is
```rust
let branches_str = if self.number_of_branches == 1 {
    String::from("1 branch")
} else if self.number_of_branches > 1 {
    format!("{} branches")
} else {
    String::new()
}
// same change for other values
```
This will result in less memory allocations.

Reply:
true, that makes sense

Reply:
adjusted

Reply:
Thanks!